website/blog/2021-11-22-sql-surrogate-keys.md

```diff
@@ -24,7 +24,7 @@ We all know one of the most fundamental rules in data is that every <Term id="ta
 
 If you are lucky, your data will come to you with a unique primary key already in place. Maybe it is an `id` generated by your internal product systems, or maybe you are pulling data in from a third party source that generates the id for you. 
 
-Sometimes, however, you find yourself in a situation where you don’t have a natural [primary key](/blog/primary-key-testing). 
+Sometimes, however, you find yourself in a situation where you don’t have a tidy [primary key](/blog/primary-key-testing). 
```

Comment:
natural has a meaning in PKs so unless we specifically mean that, just chose a synonym